superset-frontend/src/explore/components/SaveModal.tsx

```diff
@@ -145,6 +166,44 @@ class SaveModal extends React.Component<SaveModalProps, SaveModalState> {
       dashboard_title: string;
     };
 
+    if (this.props.datasource?.type === DatasourceType.Query) {
```

Comment:
Will this conditional be true for saved queries as well? Is it worth to change this to !== Table?

Reply:
i'd rather use positive logic here, and un sure we'd want the same pattern for other datasource types as we continue integrate.